src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -40,12 +40,16 @@
 
     private final List<PodVolume> volumes;
 
+    private final List<PodEnvVar> envVars;
+
     @DataBoundConstructor
-    public PodTemplate(String image, List<? extends PodVolume> volumes) {
+    public PodTemplate(String image, List<? extends PodVolume> volumes, List<PodEnvVar> envVars) {
```

Comment:
I think this is better served by keeping the old constructor (for backwards compatibility) and adding 

    @DataBoundSetter
    public void setEnvVars(...